python/ray/data/random_access_dataset.py

```diff
@@ -56,8 +57,13 @@ def __init__(
                 self._upper_bounds.append(b[1])
 
         logger.info("[setup] Creating {} random access workers.".format(num_workers))
+        ctx = DatasetContext.get_current()
+        if ctx.scheduling_strategy is not DEFAULT_SCHEDULING_STRATEGY:
```

Comment:
use `!=`?

```
>>> 'a' is not 'a'
<stdin>:1: SyntaxWarning: "is not" with a literal. Did you mean "!="?
False
```

Reply:
Oof sorry, a holdover from the `None` check